### PR TITLE
I added my files to join in the opensource contribution Anime's Art

### PR DESCRIPTION
**Pull request by Emmykeys101**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

*Optional: Edit this to tell us what you did!*

## Pull request checklist

- [ ] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [ ] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [ ] Your pull request contains `index.html` and `styles.css`.
	- [ ] Your html file is named exactly: `index.html`
	- [ ] Your css file is named exactly: `styles.css`
- [ ] Your pull request your `meta.json` with:
	- [ ] `githubHandle`: being your unique github user name,
	- [ ] `artName` : name of your animation
- [ ] Your submission includes **at least one animation**.
- [ ] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

**Review by stilt0n — CHANGES_REQUESTED** [review]
Your css file looks like it's empty. Was there supposed to be something there?

File: Art/Emmykeys101-animeArt/style.css
[the remark this answers is not in the document]
Comment (Emmykeys101) [reply Art/Emmykeys101-animeArt/style.css:1]: I will update it shortly please give me somtime

**Review by Emmykeys101 — COMMENTED** [review]
I think have been able to make some changes

File: Art/Emmykeys101-animeArt/index.html
[the remark this answers is not in the document]
Comment (Emmykeys101) [reply Art/Emmykeys101-animeArt/index.html:1]: Please check if any of the issues are resolved already let me know. am still new with PR so i might have missed somethings up. thanks

File: Art/Emmykeys101-animeArt/style.css
[the remark this answers is not in the document]
Comment (LaurelineP) [reply Art/Emmykeys101-animeArt/style.css:1]: Aloha :)
Your CSS file name is still the same and does not reflect the readme instructions on this.
- [ ] Please rename your file CSS ( then don't forget to get into your HTML in order to update the css file name

**Comment by LaurelineP** [comment]
@Emmykeys101
Aloha, thanks for your reactivity. The PR still has issues:
- incorrect css file name > hence the HTML link tag will need to be updated accordingly

File: Art/Emmykeys101-animeArt/style.css
[the remark this answers is not in the document]
Comment (Emmykeys101) [reply Art/Emmykeys101-animeArt/style.css:1]: Oops! just noticed just a "s" missing i guess so sneaky yeah will update that thanks!

**Review by stilt0n — APPROVED** [review]
This all looks good now. Thanks for making the changes.

**Pull request merged**